src/reducers/MapReducer.js

```diff
@@ -83,6 +101,12 @@ const filterSites = ({sites, filterResultsScrutinyLoose}, updatedFilterKeys) =>
         let priceMatch = false;
         let forestMatch = false;
 
+        const siteIsInView = siteIsInBoundingBox({site, boundingBox});
+
+        if(!siteIsInView){
+            return false;
+        }
```

Comment:
When filtering sites, first check to see if the site is in view. If it isn't then return `false`.